client/my-sites/email/add-mailboxes/index.tsx

```diff
@@ -192,6 +195,10 @@ const MailboxNotices = ( {
 			source,
 		} );
 
+		if ( ! selectedSite ) {
```

Comment:
What are the odds for this to occur 🤔

Reply:
Probably never, which is why I figured an error was the right choice.